src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/bookDB";
+    private static final Properties DB_PROPERTIES;
+    private static final String NOT_LOAD_DRIVE = "Can't loan JDBC driver";
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "ZXCqwe123");

```

Comment:
Don't forget to remove the passwords from the code

Reply:
don't push your creds from db to GitHub.